src/index.html

```diff
@@ -3,11 +3,310 @@
+          </li>
+          <li class="links__item">
+            <a href="#recommended" class="links__link">recommended</a>
+          </li>
+          <li class="links__item">
+            <a href="#categories" class="links__link">categories</a>
+          </li>
+          <li class="links__item">
+            <a href="#how-to-buy" class="links__link">How to buy</a>
+          </li>
+          <li class="links__item">
+            <a href="#contact-us" class="links__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+
+      <p class="menu__paragraph">
+        <a href="tel:+1 234 555-55-55" class="menu__call-link">+1 234 5555-55-55</a>
+      </p>
+      <p class="menu__paragraph">
+        <a href="tel:+1 234 555-55-55" class="menu__call-link">call to order</a>
+      </p>
+    </div>
+
+    <main class="main">
+      <section class="section main__section" id="recommended">
+        <h2 class="title title--section">
+          Recommended
+        </h2>
+
+        <div class="recommended">
+          <article class="speaker recommended__speaker">
+            <img
+              class="speaker__photo"
+              src="/images/Smart speaker.png"
+              alt="Smart speaker"
+            >
+            <h3 class="speaker__name">
+              Bose portable Smart speaker
+            </h3>
+            <p class="speaker__model">Smart home</p>
+            <p class="speaker__price">$ 399.00</p>
+          </article>
+
+          <article class="speaker recommended__speaker">
+            <img
+              class="speaker__photo"
+              src="/images/Bluetooth speaker.png"
+              alt="Smart speaker"
+            >
+            <h3 class="speaker__name">
+              SoundLink Flex Bluetooth speaker
+            </h3>
+            <p class="speaker__model">
+              Portable bluetooth
+            </p>
+            <p class="speaker__price">$ 149.00</p>
+          </article>
+
+          <article class="speaker recommended__speaker">
+            <img
+              class="speaker__photo"
+              src="/images/Bluetooth speaker II.png"
+              alt="Smart speaker"
+            >
+            <h3 class="speaker__name">
+              SoundLink Color Bluetooth speaker
+            </h3>
+            <p class="speaker__model">
+              Portable bluetooth
+            </p>
+            <p class="speaker__price">$ 129.00</p>
+          </article>
+        </div>
+
+      </section>
+
+      <section class="section main__section" id="categories">
+        <h2 class="title title--section section__title">Browse Bose products by category</h2>
+        <article class="article section__article">
+          <div class="pictures article__pictures">
+            <div class="pictures__item pictures__item--big">
+              <img
+                src="images/photos/picture-1.jpg" class="pictures__image"
+                alt="woman with earbuds"
+              >
```

Comment:
You have missed this one :)
```suggestion
              <img
                  src="images/photos/picture-1.jpg" 
                  class="pictures__image"
                  alt="woman with earbuds"
              >
```